keiko-core/src/main/kotlin/com/netflix/spinnaker/q/QueueProcessor.kt

```diff
@@ -38,13 +38,12 @@ class QueueProcessor(
   private val queue: Queue,
   private val executor: QueueExecutor<*>,
   private val handlers: Collection<MessageHandler<*>>,
-  private val activator: Activator,
+  private val activators: List<Activator>,
   private val publisher: EventPublisher,
   private val deadMessageHandler: DeadMessageCallback,
   private val fillExecutorEachCycle: Boolean = false,
   private val requeueDelay: Duration = Duration.ofSeconds(0),
-  private val requeueMaxJitter: Duration = Duration.ofSeconds(0),
-  private val enabled: Boolean = true
```

Comment:
This is replaced by `EnabledActivator` I take it

Reply:
Yes. It's only used in one place in Orca, so it'll be pretty easy to fix up.